couchbase/src/main/scala/akka/stream/alpakka/couchbase/scaladsl/CouchbaseSession.scala

```diff
@@ -53,6 +48,26 @@ object CouchbaseSession {
   def apply(bucket: Bucket): CouchbaseSession =
     new CouchbaseSessionImpl(bucket.async(), None)
 
+  /**
+   * Connects to a Couchbase cluster by creating an `AsyncCluster`.
+   * The life-cycle of it is the user's responsibility.
+   */
+  def createClusterClient(settings: CouchbaseSessionSettings)(implicit ec: ExecutionContext): Future[AsyncCluster] =
```

Comment:
Should this really be public API, if user wants low level creation shouldn't they do those two lines of creating the client themselves rather?